demo.ipynb

```diff
@@ -158,6 +158,34 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "Next, let's compare the quantile approximation to the truth using the Kullback-Leibler DIvergence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "grid1 = np.linspace(-1., 1., 200)\n",
+    "grid2 = np.linspace(-2., 2., 400)\n",
+    "grid3 = np.linspace(-3., 3., 600)\n",
+    "\n",
+    "D1 = P.kld(grid1)\n",
+    "D2 = P.kld(grid2)\n",
+    "D3 = P.kld(grid3)\n",
+    "\n",
+    "print(D1, D2, D3)"
+   ]
+  },
+  {
    "cell_type": "code",
```

Comment:
I implemented the root mean square error and added to the notebook some commentary on the meaning of the KLD.